Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -256,7 +256,7 @@ open class ChatMessageListVC:
             controller.messageController = messageController
             return controller
         }()
-
+        
```

Comment:
@jitendra-simform if possible. please remove this changes, since we are not doing any other change in this file.